.vscode/c_cpp_properties.json

```diff
@@ -3,13 +3,14 @@
         {
             "name": "Whitworth CS",
             "includePath": [
-                "\"${workspaceFolder}\"/**",
-                "\"C:/googletest/include\""
+                "${workspaceFolder}/**",
+                "/usr/local/Cellar/gcc/11.2.0_3/include/c++/11/tr1",
+                "/usr/local/Cellar/gcc/11.2.0_3/include/c++/11/x86_64-apple-darwin20"
             ],
             "defines": [
                 "_DEBUG"
             ],
-            "compilerPath": "\"C:/mingw-w64/mingw64/bin/g++.exe\"",
+            "compilerPath": "/usr/bin/clang++",
```

Comment:
I think we might need to have this file outside of source control. I had to make some changes to get it running on windows

Reply:
This file shouldn't affect how cmake works? Did you use vscode to compile. Regardless, it shouldn't be tracked.

Reply:
Nope, I was using cmake. Probably some weird disconnect between windows and mac